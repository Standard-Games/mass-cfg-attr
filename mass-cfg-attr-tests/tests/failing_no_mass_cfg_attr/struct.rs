#![deny(unused_must_use, deprecated)]
#![allow(path_statements)] // disable warning for test

#[must_use]
struct Struct();

fn main() {
    // Violation of unused_must_use if not removed by mass_cfg_attr
    Struct();
}
