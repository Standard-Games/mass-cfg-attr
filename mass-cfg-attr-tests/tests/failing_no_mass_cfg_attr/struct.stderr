error: unused `Struct` that must be used
 --> tests/failing_no_mass_cfg_attr/struct.rs:9:5
  |
9 |     Struct();
  |     ^^^^^^^^^
  |
note: the lint level is defined here
 --> tests/failing_no_mass_cfg_attr/struct.rs:1:9
  |
1 | #![deny(unused_must_use, deprecated)]
  |         ^^^^^^^^^^^^^^^
